src/service/url-replacements-impl.js

```diff
@@ -49,6 +49,28 @@ let AsyncResolverDef;
 let ReplacementDef;
 
 /**
+ * Returns a encoded URI Component, or an empty string if the value is nullish.
+ * @param {string|null|undefined}
```

Comment:
Fixed the missing `val`.

> and why undefined is allowed?

Actually, it should be `ResolverReturnDef`. Whooops.